pitop/robotics/configurations.py

```diff
@@ -0,0 +1,23 @@
+from json import load
+from pathlib import Path
+
+from pitop import Pitop
+
+
+def alex_configuration():
```

Comment:
might be better to define this as a variable instead of a function

Reply:
Definitely. Perhaps it could be an enum of stock configurations?

Reply:
```
from pitop.robotics.configs import alex_configuration
p = Pitop.from_dict(alex_configuration)